example/src/App.tsx

```diff
@@ -40,12 +33,18 @@ function App() {
         show={show}
         setShow={setShow}
       />
+      {/* <div ref={setContainer} key={width.toString()} className="h-100"> */}
```

Comment:
If you

1. undo the change below
2. uncomment this line
3. comment out the line directly below

to set up a situation where changing the width changes the key, then you'll see the "Cannot read properties of undefined" on any width change.